firmware/library/L1_Drivers/test/can_test.cpp

```diff
@@ -1,5 +1,5 @@
 #include "L0_LowLevel/LPC40xx.h"
-#include "L1_Drivers/can.hpp"
+// #include "L1_Drivers/can.hpp"
 #include "L4_Testing/testing_frameworks.hpp"
 
 // Uncomment this when can class has been created
```

Comment:
Make an empty test case and add an `EMIT_ALL_METHODS(Can);`  declaration globally in the test so that we have test coverage on the Can driver.